legacy-sdk/whirlpool/tests/utils/v2/token.ts

```diff
@@ -0,0 +1,73 @@
+import type { AnchorProvider } from "@coral-xyz/anchor";
+import { NATIVE_MINT } from "@solana/spl-token";
+import { TEST_TOKEN_PROGRAM_ID } from "../test-consts";
+import { getLiteSVM } from "../litesvm";
+
+/**
+ * Initialize the native mint (WSOL) for the regular Token Program if not already initialized.
+ * This is needed for LiteSVM testing as it doesn't automatically create the native mint.
+ */
+export async function initializeNativeMintIdempotent(provider: AnchorProvider) {
```

Comment:
is this used anywhere? i'm curious if there is a benefit of manually building account data over setting via fixture data like with the programs?

maybe this would also let us remove the `ensureNativeMintAccounts` function